### ISSUE TITLE
 Dev Jain : to be consistent in my work 

### ISSUE THREAD

**Issue by 889-dj** [asakatsuOrg/AsaKatsuProject#108]
**What is your goal?**
I will become consistent with my learnings (backend development  &. DSA) and will manage my time to be more productive 😀

### Steps to achieve the goal (not required)
  - [ ] solve one DSA question everyday 
  - [ ] read the articles and blogs 
  - [ ] make projects ( start small )
  - [ ] limit my screen time 

**Do you have a target date for this? If yes, write below**
4-5 months 

**What will you achieve by completing this goal?**
I will start applying for roles and make amazing full stack projects 😌

**Additional context**
Will try to make good habits ☺️

**Comment by avie-dev** [comment]
Thank you for committing to Asakatsu @889-dj! Hope to see your PR daily then.

**Comment by 889-dj** [comment]
Yeahh that's the plan avie to show up daily 😄

**Comment by alisamadiii** [comment]
Yeah, being consistent in work is perfect but damn it is too hard 😅
Wish you all the best @889-dj

**Comment by 889-dj** [comment]
Thanks @AliReza1083 yea agree with u hopefully I m on it and will try to continue to do better😄
